web/viewer.css

```
@@ -1373,9 +1373,10 @@ dialog :link {
   display: block;
   inset: 0;
   overflow: hidden;
-  z-index: 50000; /* should be higher than anything else in PDF.js! */
 }
-
+.grab-to-pan-grabbing {
+  z-index: 50000; /* should be higher than anything else in PDF.js*/
+}
```

Nit: This definitely needs a blank line at the end, since it's not related to the `@page` definition that follows just below.
It also seems that you've "broken" the formatting of the comment, since there should always be a space before the `*/`-part.
```suggestion
.grab-to-pan-grabbing {
  z-index: 50000; /* should be higher than anything else in PDF.js! */
}

```

---

Also, the *commit message* no longer agrees with the PR title.